jaxrs-server/src/main/java/io/micronaut/jaxrs/runtime/core/JaxRsRuntime.java

```diff
@@ -40,12 +38,6 @@ public class JaxRsRuntime {
      * @param application The application
      */
     protected JaxRsRuntime(ApplicationContext applicationContext, Application application) {
-        final Set<Object> singletons = application.getSingletons();
```

Comment:
why did you remove this?

Reply:
`Application.getSingletons()` in core is deprecated, and all it does is return `Collections.emptySet()`, so there is nothing to register.